modules/components/src/DataTable/TableToolbar/TableToolbar.js

```diff
@@ -170,10 +175,12 @@ const TableToolbar = ({
         {multipleExporters ? ( // check if we're given more than one custom exporter
           <DropDown
             aria-label={`Download options`}
-            itemToString={(i) => i.exporterLabel}
+            itemToString={(i) =>
+              typeof i.exporterLabel === 'function' ? <i.exporterLabel /> : i.exporterLabel
+            }
             items={exporterArray}
             onChange={({ exporterColumns, exporterLabel, exporterFileName, exporterFunction }) =>
-              exporterFunction(
+              exporterFunction?.(
```

Comment:
missed this codeguard last time, but useful now that functions are not mandatory